class-pinterest-for-woocommerce.php

```diff
@@ -226,6 +229,10 @@ public function init_plugin() {
 
 			$this->includes();
 
+			// Start the heartbeat.
+			$this->heartbeat = new Pinterest\Heartbeat( WC()->queue() );
```

Comment:
I know prior art here is to use relative class references, but I think we should start using imports. At a later date we can look into getting older code to match.

```suggestion
			$this->heartbeat = new Heartbeat( WC()->queue() );
```